zubhub_backend/zubhub/creators/utils.py

```diff
@@ -237,6 +237,42 @@ def activity_notification(activities, **kwargs):
         )
 
 
+def send_notification(users, contexts, template_name):
+    from .models import Setting
+
+    email_contexts = []
+    sms_contexts = []
+
+    for user, context in zip(users, contexts):
+        user_setting = Setting.objects.get(creator=user)
+
+        if user_setting.contact == Setting.WHATSAPP:
+            context.update({"phone": user.phone})
+            send_whatsapp(
+                phone=user.phone,
+                template_name=template_name,
+                ctx=context)
+
+        if user_setting.contact == Setting.EMAIL:
+            context.update({"email": user.email})
+            email_contexts.append(context)
+
+        if user_setting.contact == Setting.SMS:
+            context.update({"phone": user.phone})
+            sms_contexts.append(context)
```

Comment:
Before adding their phone number or email, verify that they have one. ```detect_mentions``` has an example of this.